tasks/win.ps1

```diff
@@ -0,0 +1,33 @@
+[CmdletBinding()]
```

Comment:
I'm guessing this is a limitation of powershell, but this doesn't seem to work on Windows <= 8.1 :(  I'm not sure how much of a deal that is - it looks like that's still a supported agent platform, but I'm also not sure what kind of powershell backbending we'd need to have to workaround or if that's worth it.

Reply:
Bumped default reconnect_timeout to 180 for older Windows platforms that are really slow to reboot. 120 wasn't enough for it to come back.